src/tls/keypair.h

```diff
@@ -844,6 +854,8 @@ namespace tls
       if (mbedtls_x509write_crt_set_serial(&sign.crt, &sign.serial) != 0)
         return {};
 
+      // TODO: macOS certificates require 825-day maximum validity
+      // (https://support.apple.com/en-us/HT210176)
```

Comment:
Let's just switch to something shorter then? We don't have any test setups that run for this long anyway.

Reply:
Done. Set to `Dec 31 23:59:59 2021 GMT`.

Reply:
I don't think we can solve this yet. The validity range of the service's cert _must_ be defined from outside (ie - config, member consensus), since the enclave cannot retrieve current time itself. If we hardcode anything shorter, we're just going to be confused when all our tests start failing in 2 and a half years. We should deal with this properly alongside service cert expiry/rotation.